socialBot.py

```diff
@@ -0,0 +1,76 @@
+import os
+from dotenv import load_dotenv, find_dotenv
+import openai
+from openai import OpenAI
+
+
+
+load_dotenv(find_dotenv())
+openai.api_key = os.getenv("API_KEY")
+
+# aimemory = "Question: In a coffee shop, is it good to yell loudly"
+# userMemory = "Yes it is appropriate if you need something very urgently"
+memory = "AI: In a coffee shop, is it appropriate to yell loudly? User: Yes it is appropriate if you need something very urgently"
+numAsks = 0
+
+# def generateResponse(memory, prompt):
```

Comment:
I'm also a massive fan of hoarding code by commenting it out, but it's pretty bad practice because its a ton of clutter. You should be using git for this. 

Create a commit with your v1 of generateResponse (e.g. git commit -am "Create a generateResponse function")
Revise the function, (git commit -am "Rename generateResponse -> ask_openai; [functionality changes go here]")

You can then reference your first commit and earlier code using git. 
```
git log
# Find your first commit
git checkout SHA(the random characters next to the commit. 
# Truncated SHA (~4+ characters) will also work
```